dhcpsnoop/dhcpsnoop.py

```diff
@@ -47,19 +47,24 @@ class CaptureThread(threading.Thread):
     Thread to sniff the network packets, sniff is a
     blocking call.
     """
-    def __init__(self, data_callback, pktcount=5, pkttimeout=5):
+    def __init__(self, data_callback, pktcount=5, pkttimeout=5, iface=None):
         threading.Thread.__init__(self)
         self.pktcount = int(pktcount)
         self.pkttimeout = int(pkttimeout)
         self.data_callback = data_callback
+        self.iface = iface
 
     def run(self):
         """
         Capture DHCP packets on the network
         """
-
-        sniff(filter="port 67 and not host 0.0.0.0", timeout=self.pkttimeout, 
-                count=self.pktcount, prn=self.data_callback, store=0)
+        LOG.debug("Sniffing on interface %s" % self.iface)
+        sniff(count=self.pktcount,
+              filter="port 67 and not host 0.0.0.0",
+              iface=self.iface,
```

Comment:
If self.iface is None does sniff default to "any"?

Reply:
Yes it does.
https://github.com/secdev/scapy/blob/master/scapy/sendrecv.py#L641-L642